### ISSUE TITLE
Android, frida.NotSupportedError: error binding to address

### ISSUE THREAD

**Issue by pandasauce** [frida/frida-core#39]
So I tried to play around with Frida on Android and the very first basic snippet of code refuses to work. The device is a rooted S4 Mini. Any ideas?

Code:

```python
import frida,sys

process = frida.get_remote_device().attach("com.android.systemui")
print process.enumerate_modules()
```

Output:

```
C:\Code\frida>python client.py
Traceback (most recent call last):
  File "client.py", line 8, in <module>
    device.attach("com.android.systemui")
  File "C:\Compilers\Python27\lib\site-packages\frida-4.4.0-py2.7-win-amd64.egg\
frida\core.py", line 78, in attach
frida.NotSupportedError: error binding to address: No such file or directory
```

Relevant method in ```core.py```:

```python
    def attach(self, target):
        return Session(self._impl.attach(self._pid_of(target)))
```

**Comment by oleavr** [comment]
Thanks for reporting! Is `frida-server` running as root and does `/data/local/tmp` exist?

**Comment by pandasauce** [comment]
Yes to both.

**Comment by oleavr** [comment]
Let's assume this is fixed in the recently released 4.5.1 (please reopen if not).
